ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2

```diff
@@ -2,17 +2,30 @@
 {% if management_api_gnmi is arista.avd.defined %}
 !
 management api gnmi
-{%     for vrf in management_api_gnmi.enable_vrfs | arista.avd.natural_sort %}
-{%         if vrf == 'default' %}
+{%     if management_api_gnmi.enable_vrfs is not arista.avd.defined %}
    transport grpc default
-{%         else %}
+{%         if management_api_gnmi.notification_timestamp.send_time is arista.avd.defined(true) %}
+      notification timestamp send-time
+{%         endif %}
+{%         if management_api_gnmi.access_group is arista.avd.defined %}
+      ip access-group {{ management_api_gnmi.access_group }}
+{%         endif %}
+{%     else %}
+{%         for vrf in management_api_gnmi.enable_vrfs | arista.avd.natural_sort %}
+{%             if vrf == 'default' %}
+   transport grpc default
+{%             else %}
    transport grpc {{ vrf }}
+{%             endif %}
 {%             if management_api_gnmi.enable_vrfs[vrf].access_group is arista.avd.defined %}
       ip access-group {{ management_api_gnmi.enable_vrfs[vrf].access_group }}
+{%             endif %}
+{%             if management_api_gnmi.notification_timestamp.send_time is arista.avd.defined(true) %}
+      notification timestamp send-time
 {%             endif %}
       vrf {{ vrf }}
```

Comment:
This will print ```vrf default```. Shouldn't we need to do a check here if vrf != default then only print this command